memiavl/db_test.go

```diff
@@ -224,6 +224,8 @@ func TestLoadVersion(t *testing.T) {
 	}
 	require.NoError(t, db.WaitAsyncCommit())
 
+	require.NoError(t, db.WaitAsyncCommit())
```

Comment:
why check NoError twice?

Reply:
good catch, that's because of merge, fixed.